crit-go/cmd/decode.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"crit-go/gocrit"
```

Comment:
I believe you need to use a full path to gocrit. It should be something like github.com/checkpoint-restore/go-criu/pkg/crit